python/py-prawcore/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-prawcore
+version             2.3.0
+supported_archs     noarch
+license             BSD
+
+python.versions     37 38 39 310
+
+maintainers         nomaintainer
+
+description         ${name} is a low-level communication layer used by PRAW 4+.
+long_description    ${description}
```

Comment:
One more comment: when using `${description}` to fill in `long_description`, you need to use the argument expansion operator (because of how Tcl, the language these Portfiles are implemented in, works):

```
description         ${name} is a low-level communication layer used by PRAW 4+.
long_description    {*}${description}
```

(Note the `{*}`)

Reply:
Hi @herbygillot,

I have fixed this and will make a note of this for future use.  

Thanks,

-ranga